### ISSUE TITLE
aurelia-cli + minified kendo does not work

### ISSUE THREAD

**Issue by JeroenVinke** [aurelia-ui-toolkits/aurelia-kendoui-bridge#616]
with minified kendo.grid.min.js this gets in the bundle:
```
define("kendo/js/kendo.grid.min", [],function(){});
```
and with unminified kendo.grid.js this gets in the bundle:

![blob](https://files.gitter.im/adriatic/Aurelia-KendoUI/A69L/blob)

kendo minified (commonJS, when downloaded from NPM) works
kendo minified (AMD, when downloaded as trial or from the kendo site) fails
kendo unminified (AMD, from the kendo source ZIP of the telerik site) works

**Comment by adriatic** [comment]
Is this not an Aurelia-CLI bug?

**Comment by JeroenVinke** [comment]
Yes, but it would be nice if we could narrow down the problem so that it can be fixed more quickly

**Comment by adriatic** [comment]
No disagreement from me

**Comment by JeroenVinke** [comment]
This may be expected behavior because we are treating Kendo as a package, but it is a collection of individual named modules. Will update the docs when we can handle that better in the CLI

**Comment by adriatic** [comment]
How can this be handled better in cli? A special case for collections of individually named modules? How frequent is this situation?

Would it be better to provide an utility that makes a package from a collection of individually named modules?

**Comment by JeroenVinke** [comment]
After the CLI respects `paths` in aurelia.json, I bet it'll work by pointing each named module to the correct location. But specifying all files is not very optimal, so perhaps we can do the same as JSPM, to allow for globs (`"kendo.*": "node_modules/kendo/kendo.*.min.js"`.

**Comment by adriatic** [comment]
Yes, as the latter ought to be just an implicit invocation of the `kendo.*.min.js` to a "private list" of paths.
